js/stdlib/ts/ALGO.ts

```diff
@@ -1763,8 +1763,8 @@ const connectAccount = async (networkAccount: NetworkAccount): Promise<Account>
           getOutput: (async <X extends CBR_Val>(o_mode:string, o_lab:string, o_ctc:ALGO_Ty<X>, o_val:X): Promise<X> => {
             void(o_mode);
             void(o_lab);
-            void(o_val);
-            return o_ctc.defaultValue;
```

Comment:
This is what I meant. You are now letting an undefined get in the runtime

Reply:
Ideally you'd make a test to show the problem, such as a remote returning a tuple

Reply:
oh, i was confused about that as well, but forgot that I had changed this. thanks for pointing it out.

Reply:
Noice